docs/index.mdx

```diff
@@ -7,7 +7,7 @@ description: "If you want to make your software applications more sustainable an
 
 Do you want to be a part of the green software movement?
 
-If you want to make your software applications more sustainable and build a career in green software, then this training is for you. It will teach you how to build, maintain and run greener applications irrespective of the application domain, industry, organization size or type, programming language, or framework.
+If you want to make your software applications more sustainable and build a career in green software, then this training is for you. It will teach you how to build, maintain, and run greener applications, regardless of the application domain, industry, organization size or type, programming language, or framework.
```

Comment:
somehow, I would not add extra comma here after maintain and applications, but I like regardless more than irrespective.  .

Reply:
Sweet! Let's add regardless and remove the comma @franziska-warncke

Reply:
Done @franziska-warncke , happy for you to check and merge accordingly